src/bpftrace.cpp

```diff
@@ -317,50 +315,48 @@ std::vector<uint64_t> BPFtrace::get_arg_values(std::vector<Field> args, uint8_t*
         switch (arg.type.size)
         {
           case 8:
-            arg_values.push_back(*(uint64_t*)(arg_data+arg.offset));
+            arg_values.emplace_back(new PrintableInt(*(uint64_t*)(arg_data+arg.offset)));
```

Comment:
It'd be a bit safer to change these `emplace_back(new...)`s to `push_back(make_unique<>(...))`